build/GlobalAttrExclusions.txt

```diff
@@ -0,0 +1,4 @@
+// These attributes should be excluded from ApiCompat checks.
```

Comment:
This file tells the ApiCompat analysis to ignore these two compiler-generated attributes. This is a repo-wide fix as opposed to what we did the last time we ran into this: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868#issuecomment-1033250875